src/api/todos.ts

```diff
@@ -0,0 +1,39 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = async (userId: number) => {
+  const response = await client.get<Todo[]>(`/todos?userId=${userId}`);
+
+  return response;
+};
+
+export const postTodos = async (userId: number, todo: Todo | null) => {
+  const response = await client.post<Todo[]>(`/todos?userId=${userId}`, todo);
```

Comment:
цей запит наче вертає нову додану сутність
```suggestion
  const response = await client.post<Todo>(`/todos?userId=${userId}`, todo);
```